share/rocm/cmake/ROCMConfigVersion.cmake

```diff
@@ -0,0 +1,51 @@
+# This is a basic version file for the Config-mode of find_package().
+# It is used by write_basic_package_version_file() as input file for configure_file()
+# to create a version-file which can be installed along a config.cmake file.
+#
+# The created file sets PACKAGE_VERSION_EXACT if the current version string and
+# the requested version string are exactly the same and it sets
+# PACKAGE_VERSION_COMPATIBLE if the current version is >= requested version,
+# but only if the requested major version is the same as the current one.
+# The variable CVF_VERSION must be set before calling configure_file().
+
+
+set(PACKAGE_VERSION "0.6.0")
```

Comment:
This seems like a duplicate to what `write_basic_package_version_file` generates. Why do we have this defined twice? Also the version should be defined in CMakeLists.txt not in this file as well(we should have one place to define the version).

Reply:
ROCMConfigVersion.cmake is the file generated by `write_basic_package_version_file`, included here only for convenience in automatic downloads of rocm-cmake (as used by almost all math libraries). By including the version file, the version can be checked and the files included directly from the download. However, it does duplicate the version information and is unrelated to the purpose of this PR and could be removed.

Reply:
> included here only for convenience in automatic downloads of rocm-cmake (as used by almost all math libraries)

But the math libraries should do install to get the version file. 

> However, it does duplicate the version information

The version should be defined in one place. We could define the version in ROCMVersion.cmake and include that in the CMakeLists.txt and ROCMVersionConfig.cmake can include it. 

> and is unrelated to the purpose of this PR and could be removed.

It would be better to put this a seperate PR.